larpandoracontent/LArVertex/MvaVertexSelectionAlgorithm.cc

```diff
@@ -614,12 +615,18 @@ std::string MvaVertexSelectionAlgorithm<T>::GetInteractionType() const
     PANDORA_THROW_RESULT_IF(STATUS_CODE_SUCCESS, !=, PandoraContentApi::GetList(*this, m_caloHitListName, pCaloHitList));
 
     // ATTN Assumes single neutrino is parent of all neutrino-induced mc particles
-    LArMCParticleHelper::MCContributionMap nuMCParticlesToGoodHitsMap;
+    LArMCParticleHelper::MCContributionMap targetMCParticlesToGoodHitsMap;
     LArMCParticleHelper::SelectReconstructableMCParticles(pMCParticleList, pCaloHitList, LArMCParticleHelper::PrimaryParameters(),
-        LArMCParticleHelper::IsBeamNeutrinoFinalState, nuMCParticlesToGoodHitsMap);
+        LArMCParticleHelper::IsBeamNeutrinoFinalState, targetMCParticlesToGoodHitsMap);
+
+    if (m_testBeamMode)
```

Comment:
Should this be an `if (m_testBeamMode) {} else {}` approach, with the IsBeamNeutrinoFinalState check in the else block, rather than before the if?

Reply:
Hi, Yeah that approach would work just as well.  In theory it will make no difference as there should only ever be either a triggered beam particle or a neutrino in a given event, but it's probably cleaner to divide the modes up as you say.